src/TheDungeon.java

```diff
@@ -104,10 +107,15 @@ public static void main(String[] argument)
         // Game loop
         while (running) 
         {
-            // Main enemy 
-            Enemy villain = new Enemy();
+            // Main enemy
+            Enemy villain = new Zombie();
```

Comment:
Why is this a `Zombie` first?

Reply:
I had to choose a type of Enemy in case the random value give it an error, Zombie are the most common monster no ? :)